pkg/controller/build/build_controller.go

```diff
@@ -39,6 +39,27 @@ const succeedStatus string = "Succeeded"
 const namespace string = "namespace"
 const name string = "name"
 
+// error message
+const (
```

Comment:
@xiujuan95 I think we should use IOTA from golang to do a simple increase, basically every time you add a new const, IOTA will generate the new number for you, see https://play.golang.org/p/Kjc5Nw3JI6Y

Reply:
Done!